nemo/collections/nlp/modules/common/common_utils.py

```diff
@@ -61,8 +61,6 @@ def get_pretrained_lm_model(
             model, default_checkpoint_file = get_megatron_lm_model(
                 config_file=config_file, pretrained_model_name=pretrained_model_name
             )
-            if not checkpoint_file:
-                checkpoint_file = default_checkpoint_file
         else:
```

Comment:
Why are you removing this? I was referring to https://github.com/NVIDIA/NeMo/blob/candidate/nemo/collections/nlp/modules/common/megatron/megatron_utils.py#L90 when was talking about an extra download call. Let's leave it for a separate PR though.

Reply:
Sorry sneaked in here from another branch.